### DISCUSSION TITLE
Issue in test suite

### DISCUSSION THREAD

**Discussion by PieterVoet** [membrane/api-gateway#1324] [General]
Hi all,

I submitted a PR ( https://github.com/membrane/api-gateway/pull/1314 ) which reports a failure in the 'Automated tests'
While looking into that , I found out that the issue already occured before my PR...  These steps already show the failure :
-  git clone https://github.com/membrane/api-gateway.git
-  mvn package -DskipTests
-  mvn test

```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 64.31 s <<< FAILURE! -- in com.predic8.membrane.core.transport.http.ServiceInvocationTest
[ERROR] com.predic8.membrane.core.transport.http.ServiceInvocationTest.testInterceptorSequence -- Time elapsed: 64.30 s <<< ERROR!
org.apache.commons.httpclient.NoHttpResponseException: The server localhost failed to respond
	at org.apache.commons.httpclient.HttpMethodBase.readStatusLine(HttpMethodBase.java:1976)
	at org.apache.commons.httpclient.HttpMethodBase.readResponse(HttpMethodBase.java:1735)
	at org.apache.commons.httpclient.HttpMethodBase.execute(HttpMethodBase.java:1098)

```
I'm not sure why the failure occurs, but I hope it's not blocking my PR from being reviewed..

Thanks.

**Comment by PieterVoet** [comment]
OK, I just saw https://github.com/membrane/api-gateway/pull/1325 was created for this.

**Comment by PieterVoet** [comment]
So, my PR remains in 'Automated tests In progress — This check has started...'  forever...  Is somebody looking into this ?
Thanks

**Comment by predic8** [comment]
Fixed in 6.0.0
